packages/template-cli/templates/src/ssr/.npmrc

```diff
@@ -1,2 +1 @@
-registry=https://pkgs.dev.azure.com/amido-dev/Amido-Stacks/_packaging/stacks-webapp-feed/npm/registry/
-always-auth=true
```

Comment:
I would remove this file completely from the ssr app - as all the packages there will be public so it doesn't need creds to pull them 
and we are never going to be pushing ssr as a package to npm